protocol/communities_messenger_test.go

```diff
@@ -4225,3 +4225,37 @@ func (s *MessengerCommunitiesSuite) TestIsDisplayNameDupeOfCommunityMember() {
 	s.Require().NoError(err)
 	s.Require().False(result)
 }
+
+func (s *MessengerCommunitiesSuite) TestUpdateGrantsForControlledCommunities() {
```

Comment:
Please run this test with -count=1000, at first sight, it seems to be a good candidate for flakiness.